packages/encrypted-blockstore/src/store-remote.ts

```diff
@@ -116,16 +116,22 @@ export class RemoteMetaStore extends MetaStoreBase {
     return this.handleByteHeads(byteHeads, branch)
   }
 
-  async save(meta: DbMeta, branch: string = 'main') {
+  async save (meta: DbMeta, branch: string = 'main') {
+    console.log("SAVING")
     const bytes = new TextEncoder().encode(this.makeHeader(meta))
```

Comment:
I think this whole thing might be double encoded, leading to some of the issues. Maybe something to form an opinion on about over time.

Reply:
ah that could be the issue - let me try it...